### ISSUE TITLE
Thompson Submachine Gun

### ISSUE THREAD

**Issue by ghost** [studio-minus/ppg-bugs-and-requests#2809]
### Feature description

Average Thompson M1A1 submachine gun, chambered in .45 ACP

### Feature purpose

We do not have SMG's chambered in .45 ACP, only 9mm

### Additional context

![m1a1](https://user-images.githubusercontent.com/104321411/226172531-ddce57cb-9dcb-48a3-b383-9f068374155c.png)
possible sprite

**Comment by Drogo-man** [comment]
The G1 is .45 acp

**Comment by ghost** [comment]
> The G1 is .45 acp

The PPG snippets doesn't even have a .45 acp casing in here

**Comment by Drogo-man** [comment]
> > The G1 is .45 acp
> 
> The PPG snippets doesn't even have a .45 acp casing in here

Yeah because they are too similarly sized to make different shell casings since the game uses a "pixel art" artstyle.

**Comment by ghost** [comment]
> > > The G1 is .45 acp
> > 
> > 
> > The PPG snippets doesn't even have a .45 acp casing in here
> 
> Yeah because they are too similarly sized to make different shell casings since the game uses a "pixel art" artstyle.

Well ingame it is slightly larger than 9mm but how are you sure it is .45 ACP and not .40 or 10mm?

**Comment by Drogo-man** [comment]
The real life counterpart is .45 ACP

**Comment by ghost** [comment]
The real life has other versions chambered in other rounds

**Comment by Drogo-man** [comment]
It's the most common caliber, at the caliber it was designed for.

**Comment by ghost** [comment]
Doesn't mean that other calibers are invalid

**Comment by Drogo-man** [comment]
I bet someone has made 22lr or 9mm Thompson before, therefore this recommendation is useless.

**Comment by ghost** [comment]
The Thompson was only produced in .45 cal, other chamber versions are unofficially modified

**Comment by Drogo-man** [comment]
You get my point, the kriss vector was originally made in .45, but then later converted to other calibers.

**Comment by ghost** [comment]
So? Just because something was made in that caliber first doesn't mean it will be the version in game

**Comment by Drogo-man** [comment]
This isnt worth my time, having idiotic online arguments over fictional firearm calibers.

**Comment by ghost** [comment]
Good for you

**Comment by thumbthumb9** [comment]
just get a mod 👍

**Comment by ghost** [comment]
> just get a mod 👍

you can apply this statement to 90% of things here

**Comment by somera139** [comment]
just to know there is only 4 bullets casings in game: pistol ones, rifle ones, sniper ones and shotgun shell casings

**Comment by somera139** [comment]
so calibers don't matter
